### PR TITLE
Calendar WIP. Fixes #11

### LINKED ISSUE #11: A Calendar on the homepage, right of the second carousel.
See [IIT Bombay's page for reference] (http://ecell.in).

### PR DESCRIPTION
**Pull request by Pk13055**
Can you check why it isn't rendering?

### CODE REVIEW

**Comment by Pk13055** [comment]
#11

File: Backend_Repo/website/template/website/index.html [hunk 1]
```diff
@@ -1,52 +1,60 @@
 {% extends "website/base.html" %} {% block main %}{% load staticfiles %}
+{% block links %}
+<script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.18.1/locale/en-gb.js"></script>
+<link href="https://cdnjs.cloudflare.com/ajax/libs/fullcalendar/3.4.0/fullcalendar.min.css" type="text/css" rel="stylesheet">
+<link href="https://cdnjs.cloudflare.com/ajax/libs/fullcalendar/3.4.0/fullcalendar.print.css" type="text/css" rel="stylesheet">
```
Comment (nemani) [inline Backend_Repo/website/template/website/index.html:5]: Use only one CSS?

File: Backend_Repo/website/template/website/index.html [hunk 2]
```diff
@@ -1,52 +1,60 @@
 {% extends "website/base.html" %} {% block main %}{% load staticfiles %}
+{% block links %}
+<script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.18.1/locale/en-gb.js"></script>
```
Comment (nemani) [inline Backend_Repo/website/template/website/index.html:3]: This is a locale file.
Not Needed.
Just use https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.18.1/moment.js instead

File: Backend_Repo/website/template/website/index.html (hunk 1 above)
Comment (Pk13055) [reply Backend_Repo/website/template/website/index.html:5]: Change it and push yeah? Too lazy :P

**Comment by nemani** [comment]
After changing it, there's other problems with css. Please help with that.
:p

**Pull request closed without merging**